projects/1.16/assets/zombie-extreme/zombie-extreme/lang/zh_cn.json

```
@@ -0,0 +1,408 @@
+{
+  "advancements.alternateworldadvancement.descr": "核战，人类的终末……",
+  "advancements.alternateworldadvancement.title": "死海",
+  "advancements.anddoesnotbreakwithoutwork.descr": "制作钛镐",
+  "advancements.anddoesnotbreakwithoutwork.title": "获得钛镐",
+  "advancements.darkpearl.descr": "熔炼一个暗金属锭",
+  "advancements.darkpearl.title": "暗珍珠",
+  "advancements.fleshofthe_infected.descr": "寻找远古血肉",
+  "advancements.rubies.descr": "挖红宝石",
+  "advancements.rubies.title": "红宝石！",
+  "advancements.titanignot.descr": "熔炼钛锭",
+  "advancements.titanignot.title": "获得钛锭",
+  "advancements.titanpower.descr": "权力在手",
+  "advancements.titanpower.title": "钛之力！",
+  "advancements.whatam_idoing.descr": "合成血肉锭",
+  "advancements.whatam_idoing.title": "我在做什么......？",
+  "advancements.whatis_that.descr": "制作血肉镐",
+  "advancements.whatis_that.title": "那是什么？",
+  "advancements.zombie_extreme_begin.descr": "感谢您的参与",
+  "advancements.zombie_extreme_begin.title": "僵尸扩展",
+  "biome.zombie_extreme.dead_sand": "死沙",
+  "block.zombie_extreme.alternatelog": "焦枯橡木原木",
+  "block.zombie_extreme.alternateoregold": "焦金矿",
+  "block.zombie_extreme.alternateoreiron": "焦铁矿",
+  "block.zombie_extreme.alternateoreradium": "焦镭矿",
+  "block.zombie_extreme.alternateoreruby": "焦化红宝石矿",
```

```suggestion
  "block.zombie_extreme.alternateoreruby": "焦红宝石矿",
```
或将所有`焦xx`且原文为`Scorched xx`的名称修改为`焦化xx`

如选择译为`焦xx`，则需将下方`焦化石头`修改为`焦石头`